src/types/auth.ts

```diff
@@ -17,3 +17,11 @@ export type UserSignupRequest = Record<
     isComplete: boolean;
   }
 > & { themePreference: { value: string[]; isComplete: boolean } };
+
+export interface nicknameResponseType {
+  uniqueNickname: boolean;
+}
+
+export interface nicknameRequestType {
+  nickname: string;
+}
```

Comment:
```suggestion
export interface NicknameResponseType {
  uniqueNickname: boolean;
}

export interface NicknameRequestType {
  nickname: string;
}
```

파스칼로 갑시다~

닉네임의 리스폰스의 필드명이 uniqueNickname인건 서버에서 저렇게 내려주기 때문이겠죠?
뭔가 해당 닉네임이 유니크한가? 라는 의미로는 `isUniqueNickname` 같은 네이밍이 더 적합해보여서요 !